integration_test.go

```diff
@@ -76,6 +74,33 @@ func (tc *testClient) SetMainById(id string) {
 	}
 }
 
+// For each matching deploy, ensure
+//   1. It's running
+//   2. The result of performing a GET query on 'path' is as expected
+func (tc *testClient) checkDeployBody(deployId string, path string, expected string) {
```

Comment:
This name is a bit weird to me. Maybe it could be checkDeployGet() or checkDeployGetResult()?